README.md

```diff
@@ -8,10 +8,19 @@
 
 ## Introduction
 
-Supply Chain Goat follows the tradition of existing *Goat projects (e.g. OWASP Web Goat). It provides a training ground to practice implementing countermeasures specific to the software supply chain. StepSecurity defines a supply chain attack as an attack that tries to hijack software that you produce or consume.
+Supply Chain Goat follows the tradition of existing *Goat projects (e.g. OWASP Web Goat). It provides a training ground to practice implementing countermeasures specific to the software supply chain. 
+
+StepSecurity defines a supply chain attack as an attack that tries to hijack software that you produce or consume. [OSSF Scorecards](https://github.com/ossf/scorecard) recommends using the StepSecurity tool to remediate some of the checks. To learn more about Software Supply Chain Security, click [here](https://github.com/step-security/supply-chain-goat/blob/main/SoftwareSupplyChain.md)
```

Comment:
After thinking more, I think let us remove this line (`To learn more about Software Supply Chain Security, click [here](https://github.com/step-security/supply-chain-goat/blob/main/SoftwareSupplyChain.md)`) and the associated page for the time being. I think the goal here is to get reader to do the tutorials to learn, and not by reading.

Reply:
@arjundashrath this one is still pending.

Reply:
okay sir, I just noticed it, will do